.github/workflows/build.yml

```
@@ -1,43 +1,24 @@
 name: Build Docker Image
-
-on: push
-
+on:
+  pull_request:
+    branches:
+      - main
+    types: 
+      - opened
+      - synchronize
+      - reopened
 jobs:
   build:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 0
-
-      - name: Set up Google Cloud Credentials
-        uses: google-github-actions/auth@v0
-        with:
-          credentials_json: ${{ secrets.GOOGLE_HOPRASSOCIATION_CREDENTIALS_REGISTRY }}
-
-      - name: Set up Google Cloud SDK
-        uses: google-github-actions/setup-gcloud@v0
+      - name: Build and push
```

Push is set to `false`, so the step name should probably by only `Build`:
```suggestion
      - name: Build
```

---

Thanks, issues with Copy&Paste ;)